tests/e2e-ui-tests/.eslintrc

```diff
@@ -0,0 +1,43 @@
+{
+  "env": {
+    "node": true,
+    "es6": true,
+    "mocha": true,
+    "protractor": true
+  },
+  "parserOptions": {
+    "ecmaVersion": 6
+  },
+  "extends": "eslint:recommended",
+  "rules": {
+    "array-callback-return": "warn",
+    "brace-style": ["error", "1tbs"],
+    "complexity": ["warn", 20],
+    "eqeqeq": "error",
+    "guard-for-in": "error",
+    "indent": ["error", 2],
+    "linebreak-style": ["error", "unix"],
+    "no-array-constructor": "error",
+    "no-console": "warn",
+    "no-lonely-if": "warn",
+    "no-loop-func": "warn",
+    "no-mixed-spaces-and-tabs": ["error"],
+    "no-nested-ternary": "error",
+    "no-spaced-func": "error",
+    "no-trailing-spaces": "error",
+    "semi": ["error", "always"],
+    "space-before-blocks": "error",
+    "space-before-function-paren": ["error", "never"],
+    "keyword-spacing": ["error"],
+    "curly": ["error", "all"],
+    "no-else-return": ["error"]
+  },
+  "globals": {
+    "angular": false,
+    "$fh": false,
+    "localStorage": false,
+    "window": false,
+    "expect": false,
+    "step": false
+  }
+}
```

Comment:
There seem to be more rules to this file than the one we're using in the other modules, do you see any of them being interesting to the rest of the repo?

Eventually I'd like to bubble the eslint config files to the root of the repo and at most extend them from each module, see related [eslint docs](https://eslint.org/docs/user-guide/configuring#using-a-configuration-file)